pytorch_lightning/trainer/data_loading.py

```diff
@@ -310,3 +310,25 @@ def determine_data_use_amount(self, train_percent_check: float, val_percent_chec
             self.train_percent_check = overfit_pct
             self.val_percent_check = overfit_pct
             self.test_percent_check = overfit_pct
+
+
+class PatchDataLoader(object):
```

Comment:
Why Is this needed now but wasn't before?

Reply:
It's just moved from the Trainer file